thirdparty/rr-full/plugins/itempos.pl

```diff
@@ -1,10 +1,8 @@
 #-----------------------------------------------------------
-# shellbags.pl
-# RR plugin to parse (Vista, Win7/Win2008R2) shell bags
-#
+# itempos.pl
+# 
 # History:
-#   20130514 - added checking of ShellNoRoam\Bags subkeys for WinXP
-#   20120814 - created
+#   20191111 - Added default value to $jmp if $item{extver} cannot be determined.
```

Comment:
Why remove previous history?

Reply:
This was removed in 2018 by keydet89 in the the regripper2.8 repo.